aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -77,6 +77,10 @@ Tensor arange(
     Scalar end,
     Scalar step,
     const TensorOptions& options) {
+  AT_CHECK(
+    finite(end.toDouble()) && finite(start.toDouble()),
```

Comment:
I think you want to use `isfinite` in C99/C++11. `finite` is Linux-specific

Reply:
ah, updated. would our tests catch this?

Reply:
yea, the CI caught it, Windows build failed :D